components/classes/lodging-product.schema.json

```diff
@@ -19,6 +19,11 @@
   "definitions": {
     "lodging-product": {
       "properties": {
+        "xdm:ID": {
```

Comment:
Since this is at the root, let's give it a more contextual name. "propertyID " ?

Reply:
Will do